README.md

```diff
@@ -58,7 +58,7 @@ systems that have installed Home Assistant.
 
 - **[Mosquitto broker](/mosquitto/README.md)**
 
-    An Open Source MQTT broker.
+    MQTT broker for Home Assistant.
```

Comment:
This is not a correct change. Mosquitto is not specifically made for  Home 
 Assistant; it is an MQTT broker. Please revert this change.